lib/modules/versioning/exact/index.spec.ts

```diff
@@ -0,0 +1,136 @@
+import exact from '.';
+
+describe('modules/versioning/exact/index', () => {
+  describe('isValid', () => {
+    it.each`
+      input              | expected
+      ${''}              | ${false}
+      ${'v1'}            | ${true}
+      ${'1.0.0'}         | ${true}
+      ${'any-string'}    | ${true}
+      ${'abc123'}        | ${true}
+      ${'sha256:abcdef'} | ${true}
+    `('isValid("$input") === $expected', ({ input, expected }) => {
+      expect(exact.isValid(input)).toBe(expected);
+    });
+  });
+
+  describe('isVersion', () => {
+    it.each`
+      input        | expected
+      ${null}      | ${false}
+      ${undefined} | ${false}
+      ${''}        | ${false}
+      ${'v1'}      | ${true}
+      ${'1.0.0'}   | ${true}
+    `('isVersion($input) === $expected', ({ input, expected }) => {
+      expect(exact.isVersion(input)).toBe(expected);
+    });
+  });
+
+  describe('isSingleVersion', () => {
+    it('returns true for any valid version', () => {
+      expect(exact.isSingleVersion('1.0.0')).toBe(true);
+      expect(exact.isSingleVersion('any-string')).toBe(true);
+    });
+  });
+
+  describe('isStable', () => {
+    it('returns true for any version', () => {
+      expect(exact.isStable('1.0.0-alpha')).toBe(true);
+      expect(exact.isStable('1.0.0')).toBe(true);
+    });
+  });
+
+  describe('isCompatible', () => {
+    it('returns true when version equals current', () => {
+      expect(exact.isCompatible('v1', 'v1')).toBe(true);
+    });
+
+    it('returns false when version differs from current', () => {
+      expect(exact.isCompatible('v1', 'v2')).toBe(false);
+    });
```

Comment:
Could we use two versions that would be SemVer compatible here? 

I.e.

```suggestion
    it('returns false when version differs from current', () => {
      expect(exact.isCompatible('v1.0.0', 'v1.1.0')).toBe(false);
    });
```

This makes it clear that, unlike SemVer, things aren't compatible even if they look like they should be